src/tests/ftest/pool/destroy.py

```diff
@@ -167,7 +165,9 @@ def test_destroy_single(self):
         """Test destroying a pool created on a single server.
 
         :avocado: tags=all,pr,daily_regression
-        :avocado: tags=pool,destroy,destroy_single
+        :avocado: tags=vm
+        :avocado: tags=pool
+        :avocado: tags=destroy,destroy_single
```

Comment:
While this file is being refactored, could we refactor the tags too? Personally, I think "destroy" and "destroy_single" aren't clear when taken out of the context of this file. I would recommend
```suggestion
        :avocado: tags=vm
        :avocado: tags=pool,pool_destroy
        :avocado: tags=pool_destroy_single
```
And likewise for other tests in this file. Then, all tests in this file could be ran with `Test-tag: pool_destroy`.
What's your opinion?

Reply:
That makes sense. I updated them. Thanks.